sdk/anomalydetector/ai-anomaly-detector/src/AnomalyDetectorClient.ts

```diff
@@ -109,6 +110,7 @@ export class AnomalyDetectorClient {
    */
   public detectEntireSeries(
     body: DetectRequest,
+    // eslint-disable-next-line @azure/azure-sdk/ts-naming-options
```

Comment:
I'd like to relax this rule to accept `OperationOptions`. Many a time we dont have anything else to add, and creating a new interface which would be exactly same as `OperationOptions` feels like an overkill.

@bterlson, @xirzec ?